portfolio.html

```diff
@@ -23,88 +23,90 @@
-				</div>
-				<div class="leftText">
-				<a href="#" class="portfolioText">Right Upper Box hello this is a text test as im trying to figure sometihng out</a>
+			<div class="portfolioBox">
+				<div class="background">
+				<a href="#"><img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage"></a>
+				<div class="leftText fontStyleHead">
+				<a href="#" class="portfolioText"><button>Left Upper Box hello this is a text test as im trying to figure sometihng outyes yes yes  yes yes yes yes yesbrhablgr </button></a>
 				</div>
 			</div>
-			<div class="leftBelowBox">
-				<div class="leftImage">
-				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
-				</div>
-				<div class="leftText">
-				<a href="#" class="portfolioText">Left Below Box hello this is a text test as im trying to figure sometihng out</a>
-				</div>
 			</div>
-			<div class="middleBelowBox">
-				<div class="leftImage">
-				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+			<div class="portfolioBox">
+				<div class="background">
+				<a href="#"><img src="https://images.pexels.com/photos/4406222/pexels-photo-4406222.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260" alt="Yo" class="portfolioImage"></a>
+				<div class="leftText fontStyleHead">
+				<a href="#" class="portfolioText"><button>Left Upper Box hello this is a text test as im trying to figure sometihng out</button></a>
 				</div>
-				<div class="leftText">
-				<a href="#" class="portfolioText">Left Box hellogggggg this is a text test as im trying to figure sometihng out</a>
+			</div>
+			</div>
+			<div class="portfolioBox">
+				<div class="background">
+				<a href="#"><img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage"></a>
+				<div class="leftText fontStyleHead">
+				<a href="#" class="portfolioText"><button>Left Upper Box hello this is a text test as im trying to figure sometihng out</button></a>
 				</div>
 			</div>
-			<div class="rightBelowBox">
-				<div class="leftImage">
-				<img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage">
+			</div>
+			<div class="portfolioBox">
+				<div class="background">
+				<a href="#"><img src="https://images.pexels.com/photos/5137457/pexels-photo-5137457.jpeg?auto=compress&cs=tinysrgb&dpr=2&w=500" alt="Yo" class="portfolioImage"></a>
+				<div class="leftText fontStyleHead">
+				<a href="#" class="portfolioText"><button>Left Upper Box hello this is a text test as im trying to figure sometihng out</button></a>
```

Comment:
Do you want to use a button to hold text in?

Reply:
Same situation with regards to the nav bar and making it look nice. I just couldn't get text-decoration: none to work. So this is what I came up with.